dashboard/common/db_util.py

```diff
@@ -110,9 +109,27 @@ def get_user_courses_info(username: str) -> List[Dict[str, Union[str, int, List[
                 course_enrollments[course_id]['enrollment_types'].append(enrollment_type)
             user_courses_info = list(course_enrollments.values())
     logger.info(f'User {username} is enrolled in these courses: {user_courses_info}')
+
+    course_id_to_filter = get_course_id_from_request_url(request)
+    if course_id_to_filter is not None and len(user_courses_info) != 0:
+        for course in user_courses_info:
+            if course['course_id'] != course_id_to_filter:
+                user_courses_info.remove(course)
```

Comment:
Do you give any consideration to doing this outside of `get_user_courses_info` in `get_myla_globals`? I guess I could see arguments either way.

Reply:
This function is handling the course info and i need to filter the courses based on the LTI vs standalone. 

I have separated out logic for multiple course check info to a function. Bit Simplified